applications/app/controllers/InteractiveController.scala

```diff
@@ -139,27 +127,53 @@ class InteractiveController(
     }
   }
 
+  def renderDCRAmp(ir: ItemResponse)(implicit request: RequestHeader): Future[Result] = {
+    fItemResponseToFE(Future.successful(ir)) flatMap {
+      case Left((model, blocks)) => {
+        val pageType = PageType.apply(model, request, context)
+        remoteRenderer.getAMPInteractive(wsClient, model, blocks, pageType)
+      }
+      case Right(other) => Future.successful(RenderOtherStatus(other))
+    }
+  }
+
+  def itemResponseToInteractivePickerInputData(ir: ItemResponse)(implicit
```

Comment:
There are only two arguments here. I'd be tempted to scap this method and the `InteractivePickerInputData` case class and just pass them directly?

Reply:
Done :)